### PR TITLE
add to cart

### PR DESCRIPTION
**Pull request by luyentap**
https://edu-redmine.sun-asterisk.vn/issues/8246

### CODE REVIEW

**Comment by luyentap** [comment]
ready

File: src/reducers/productReducer.js [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+import {ADD_TO_CART, PRODUCT_FETCH_SUCCESS} from "../actions/ActionTypes";
 import {combineReducers} from 'redux'
 import ProductService from "../service/ProductService";
 import React from "react";
+import {loadState} from "../common/LocalSave";
 
+//lấy giá trị(cart+total) trong localstorage -->init reducer
+let initTotal = loadState("total")||0;
+let initAddedItems = loadState("shoppingCart")||[];
+console.log("init",initAddedItems);
```
Comment (chien-pv) [inline src/reducers/productReducer.js]: bỏ console đi

**Comment by chien-pv** [comment]
commented

File: src/reducers/productReducer.js (hunk 1 above)
Comment (luyentap) [reply src/reducers/productReducer.js]: ready

**Comment by luyentap** [comment]
ready

**Pull request merged**